functions/index.js

```diff
@@ -35,27 +40,58 @@ exports.createNewTask = functions.https.onRequest((request, response) => {
         console.log(category);
         console.log(createNewTaskSprintNumber);
 
-        db.collection(fullSprintId).doc(category).set({
-                Title: title,
-                Description: des,
-                Priority: priority,
-                Difficulty: difficulty,
-                Creator: creator,
-                Assignee: assignee,
-                ET: estimatedTime,
-                Status: status,
-                Category: category,
-                LogWorkTotalTime: loggedWorkTotalTime,
-                WorkDone: workDone,
-                CreateNewTaskSprintNumber: createNewTaskSprintNumber
+        db.collection("Main").doc("RawData").get().then((doc) => {
+
+                totalDevelopmentTask = doc.data().TotalDevelopmentTask + 1;
+                totalBusinessTask = doc.data().TotalBusinessTask + 1;
+                totalMarketingTask = doc.data().TotalBusinessTask + 1;
+                totalNumberOfTask = (totalDevelopmentTask + totalBusinessTask + totalMarketingTask);
```

Comment:
totalNumberOfTask can also be db field rather than manual calculation here,

few more fields can be there in db totalCompletedTask and totalUncompletedTask

These counters can be used in future based on the need.

Reply:
totalNumberOfTask ...  actually I haven't understood what to do here
remaining I have completed